homeassistant/components/lifx/coordinator.py

```diff
@@ -75,9 +102,79 @@ def mac_address(self) -> str:
             self.device.host_firmware_version,
         )
 
+    @property
+    def label(self) -> str:
+        """Return the label of the bulb."""
+        return cast(str, self.device.label)
+
+    def get_rssi_unit_of_measurement(self) -> str | None:
+        """Return the unit of measurement for the RSSI sensor."""
+        if self.device.host_firmware_version is not None:
+            if AwesomeVersion(self.device.host_firmware_version) > RSSI_DBM_FW:
+                return SIGNAL_STRENGTH_DECIBELS_MILLIWATT
+            return SIGNAL_STRENGTH_DECIBELS
+
+        return None
+
     async def _async_update_data(self) -> None:
         """Fetch all device data from the api."""
+
+        @Throttle(timedelta(seconds=60))
+        async def _async_update_rssi() -> None:
+            """Fetch the current RSSI value if the sensor is enabled."""
+            try:
+                self.rssi = get_rssi_from_wifiinfo(
+                    await async_execute_lifx(self.device.get_wifiinfo)
+                )
+            except asyncio.TimeoutError as ex:
+                raise UpdateFailed(
+                    f"Failed to fetch RSSI value from device: {self.device.label} ({self.device.ip_addr})"
+                ) from ex
+
+        @Throttle(timedelta(seconds=60))
+        async def _async_update_hev_cycle_status() -> None:
+            """Fetch current measurement for enabled HEV sensors."""
+            try:
+                (
+                    self.hev_cycle_duration,
+                    self.hev_cycle_remaining,
+                    self.hev_cycle_last_power,
+                ) = hev_cycle_status(
+                    await async_execute_lifx(self.device.get_hev_cycle)
+                )
+            except asyncio.TimeoutError as ex:
+                raise UpdateFailed(
+                    f"Failed to fetch HEV cycle status from device: {self.device.label}() {self.device.ip_addr})"
+                ) from ex
+
+        @Throttle(timedelta(seconds=60))
```

Comment:
We try to avoid using Throttle in new code

If we need different intervals, usually we use multiple coordinators or write a custom coordinator. 

It's a bit more difficult here since we need a shared lock but the advantage is if you have multiple coordinators you don't have to worry about subscription management since the coordinators don't poll if there are no subscribers

Reply:
Strictly speaking, we don't need different intervals but then we also don't need to poll for sensor data every 10 seconds either. I'm more worried about the amount of data potentially being stored by the logger for no good reason if we poll too often.

Reply:
We probably don't need to share a lock either: that's only really needed for discovery. Once the transport is unicast UDP, the bulbs tend to respond pretty quickly and can handle up to 20 pkts/second. 

As a reference, the Tile animations generated by Photons send 320 HSBK values every 0.075 seconds. :)

Reply:
We would be better suited with a global limit of 2-4 inflight (unacknowledged) packets per endpoint, which is automatic in Photons but would have to be added to aiolifx.

Reply:
Reducing the polling frequency makes a lot of sense here. If you have a lot of bulbs the additional wifi traffic will make a difference on how much reliable your 2.4g network is.

Reply:
Agreed. And it would reduce the amount of potential state data being stored by the logger. I know it's only stored on changes, but given the potential flakiness of LIFX bulbs, the RSSI will change every polling interval for sure.